Makefile

```diff
@@ -140,7 +140,7 @@ clean:
 	find $(AVOCADO_OPTIONAL_PLUGINS) -name '*.egg-info' -exec rm -r {} +
 
 pip:
-	$(PYTHON) -m pip --version || $(PYTHON) -c "import os; import sys; import urllib; f = urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py')[0]; os.system('%s %s' % (sys.executable, f))"
+	$(PYTHON) -m ensurepip $(PYTHON_DEVELOP_ARGS) || $(PYTHON) -c "import os; import sys; import urllib; f = urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py')[0]; os.system('%s %s' % (sys.executable, f))"
```

Comment:
On older systems this always fails with `/usr/bin/python: No module named ensurepip`. How about `$(PYTHON) -m pip --version || python -m ensurepip $(PYTHON_DEVELOP_ARGS) || $(PYTHON) ...`